### DISCUSSION TITLE
Godot: Bug in instantiation of physics bodies with joint. Where to post an issue?

### DISCUSSION THREAD

**Discussion by Conz3D** [jrouwe/JoltPhysics#769] [Q&A]
How can I figgure out, if a bug is in Jolt or in the implementation in an engine? Like the Jolt Addon for Godot.
I have a very strange bug, that happens not every time, but more often with the amount Jolt has to handle.
I have two rigid bodys connected by a 6DOF Joint with limits set to 0 to use it as a fixed joint. Sometimes when I instantiate this, the attached object is falling for some frames, before the joint will get calculated. It happens not every time, but it seems correlated to the amount of other physics Jolt has to calculate (not an crazy amount).
Is this a known problem, or is this a problem with the Godot implementation. How can I help the developer of the Godot Addon to narrow this down, if this is a Godot or a Jolt problem?

There is this discussion about it in the Godot-Jolt git:
https://github.com/godot-jolt/godot-jolt/discussions/654

Inbetween I thought it would be fixed, and bec. it's bound to the complexity of the simulation, it wasn't reproduceble in a simplified scene. And maybe it's appearing early on my system, since it's not the newest one.

**Comment by jrouwe** [comment]
Let's leave the discussion in the original tracker.
